src/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/node_builders/param_vector.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "openvino/op/parameter.hpp"
+
+namespace ov {
+namespace test {
+namespace utils {
+namespace builder {
+
+ov::ParameterVector makeParams(const ov::element::Type &type, const std::vector<std::vector<size_t>> &shapes);
```

Comment:
Why cannot we remove these methods and in tests use `std::make_shared<Parameter>()` in the cycle?
@iefode ??

Reply:
@ilyachur  I am absolutely ok to remove the all builders :)

Reply:
@olpipi Let's move to use OV api instead of builders. My suggestion is to remove theam